src/mlpack/core/optimizers/sgd/sgd.hpp

```diff
@@ -135,6 +136,9 @@ class SGD
   //! Modify whether or not the individual functions are shuffled.
   bool& Shuffle() { return shuffle; }
 
+  //! Get the update policy. 
+  UpdatePolicy GetUpdatePolicy() const { return updatePolicy; }
```

Comment:
Also please add ``UpdatePolicy& UpdatePolicy() { return updatePolicy; }``, so that we can modify the policy.